content/posts/2021/reana-0.7.4.md

```diff
@@ -0,0 +1,134 @@
+  stages:
+    - name: gendata
+      dependencies: [init]
+      scheduler:
+        scheduler_type: 'singlestep-stage'
+        parameters:
+          events: {step: init, output: events}
+          gendata: {step: init, output: gendata}
+          outfilename: '{workdir}/data.root'
+        step:
+          process:
+            process_type: 'interpolated-script-cmd'
+            script: root -b -q '{gendata}({events},"{outfilename}")'
+          publisher:
+            publisher_type: 'frompar-pub'
+            outputmap:
+              data: outfilename
+          environment:
+            environment_type: 'docker-encapsulated'
+            image: 'reanahub/reana-env-root6'
+            imagetag: '6.18.04'
+            resources:
+              - compute_backend: kubernetes
++             - kubernetes_memory_limit: '8Gi'
+```
+### Custom memory limit for CWL workflow
+
+We can set `kubernetes_memory_limit` in every step under `hints.reana`.
+
+```diff
+  ...
+  steps:
+    gendata:
+      run: gendata.cwl
+      in:
+        gendata_tool: gendata_tool
+        events: events
+      hints:
+        reana:
+          compute_backend: kubernetes
++         kubernetes_memory_limit: '8Gi'
+      out: [data]
+```
+
+### Custom memory limit administration
+
+REANA cluster administrators can configure the default memory limit for Kubernetes compute backend user job containers by setting
+[kubernetes_jobs_memory_limit](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value and specifying the desired value accordingly.
+
+Also, administrators can set the maximum custom memory limit value that users can assign to their job containers via
+`kubernetes_memory_limit` hint in `reana.yaml` by changing the
+[kubernetes_jobs_max_user_memory_limit](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value.
+
+Please refer to
+[Kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
+for expected memory values and note that exceeding these limits will terminate user job containers.
+
+### reana-client improvements
+
+REANA 0.7.4 command line client introduces a wildcard matching patterns support
+for `ls` and `download` commands. This change brings useful filtering
+capabilities which might be especially useful for large workflows with many files.
+The release also includes improvements for `reana-client` workflow validation
+of input parameters and environment images.
+
+Enjoy!
+
+See also:
+- [REANA 0.7.4 release notes](https://github.com/reanahub/reana/releases/tag/0.7.4)
+- [REANA installation guide](https://docs.reana.io/administration/deployment/deploying-at-scale/)
+- [REANA custom memory limits for Kubernetes](https://docs.reana.io/advanced-usage/compute-backends/kubernetes/#custom-memory-limit)
+- [Kubernetes official documentation for memory limits](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
```

Comment:
Perhaps we should remove the link to the Kubernetes official documentation here, as it could confuse some users regarding limits vs requests... Since REANA does things a bit differently.  So the Kubernetes docs will be simply linked to in the "admin" section", but otherwise not advertised in the footer links or in the "user" section, for the sake of simplicity.